src/manifest.json

```diff
@@ -1,11 +1,54 @@
 {
-    "manifest_version":2,
-    "version":"1.0",
-    "name":"Mozilla VPN integration",
-    "author": "Firefox",
+    "manifest_version": 2,
+    "version": "1.0",
+    "author": "Mozilla",
+    "name": "Mozilla VPN Web Extension (TODO Localize)",

```

Comment:
How would we localize this ? 😅

Reply:
https://github.com/mozilla-extensions/secure-proxy/blob/0e09673d9ecbc9cedbe02a0108062b795cf58543/src/manifest.json#L3 

But with Fluent... I am not sure (yet).